apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -1018,6 +1019,12 @@ export class MarkdownDocumenter {
       }
     }
 
+    if (ApiReadonlyMixin.isBaseClassOf(apiItem)) {
+      if (apiItem.isReadonly) {
+        section.appendNodeInParagraph(new DocCodeSpan({ configuration, code: 'readonly' }));
```

Comment:
I threw a static readonly property into docclass1 in api-documenter-tests and got this output:

```
|  [staticReadonlyThing](./api-documenter-test.docclass1.staticreadonlything.md) | <code>static</code><code>readonly</code> | boolean |  |
```

I believe this would be satisfactory

Reply:
As we discussed, let's pull out the api-documenter changes and I'll include them as part of https://github.com/zelliott/rushstack/pull/2.

Reply:
Believe this has been resolved